### PR TITLE
Add media query to the style.css file

### PR DESCRIPTION
**Pull request by Yidnekachew-cmd**
In this branch, I added a media query to the style.css file. I used a min-width 768px to make the portfolio responsive for the desktop.  In addition, this branch also has background images and thumbnails.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,124 +73,257 @@ <h1 class="intro">I’m Yidnekachew Glad to see you!</h1>
...
-                <div class="left-block2">
-                    <div class="text2">
-                        <h2 class="title2">Multi-Post Stories</h2>
-                        <ul class="frame2">
-                            <li class="client2">CANOPY</li>
-                            <li class="role2">Back End Dev</li>
-                            <li class="year2">2015</li>
+                <div class="left-block">
+                    <div class="text">
+                        <h2 class="title">Multi-Post Stories</h2>
```
Comment (topeogunleye) [inline index.html:146]: - [ ] I kindly suggest that you add the grey circular icons present in the Figma design to your own design.

| Your Design  | Figma Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 2]
```diff
@@ -60,124 +73,257 @@ <h1 class="intro">I’m Yidnekachew Glad to see you!</h1>
...
-                    <div class="action2">
-                        <div class="normal-button2">
+                    <div class="action">
+                        <div class="normal-button">
                             <button type="button">See Project</button>
                         </div>
                     </div>
                 </div>
+                </div>
+                <div class="desktop">
```
Comment (topeogunleye) [inline index.html:170]: - [ ] I kindly suggest that you make sure that your website is properly responsive to look like the Figma design.
20% of your page requires an horizontal scroll bar to be viewed. Please see my video below for more explanation.

image


https://www.loom.com/share/c56e74d7e441418f8102b08b9dd9e602

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @Yidnekachew-cmd, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Yidnekachew-cmd) [reply index.html:146]: I have inserted bullets as you suggested me. Thank you!

Comment (topeogunleye) [reply index.html:146]: @Yidnekachew-cmd , That's great!

File: index.html (hunk 2 above)
Comment (Yidnekachew-cmd) [reply index.html:170]: I did the changes to my project. Thank you.

Comment (topeogunleye) [reply index.html:170]: @Yidnekachew-cmd, you're welcome

**Review by Olamarx — APPROVED** [review]
Hi @Yidnekachew-cmd, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linters passed ✔️ 
- Git and GitHub flow 👍🏼 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**